src/ui/public/vis/request_handlers/courier.js

```diff
@@ -74,52 +63,40 @@ const CourierRequestHandlerProvider = function (Private, courier, timefilter) {
         return timefilter.get(searchSource.get('index'), timeRange);
       });
 
-      removeSearchSourceParentTimefilter(requestSearchSource);
-
-      if (queryFilter && vis.editorMode) {
-        searchSource.set('filter', queryFilter.getFilters());
-        searchSource.set('query', appState.query);
-      }
+      requestSearchSource.set('filter', filters);
```

Comment:
This does actually not work, since we will overwrite the timefilter above on that search source and thus lose our time filter.

Reply:
thats for finding this ..... not sure when i lost this, i have the timeFilterSearhSource created i just don't use it anywhere | ... updated

Reply:
Is this resolved? Best I can tell this is working.

Reply:
yup